app/views/recipe/show.html.erb

```diff
@@ -0,0 +1,60 @@
+<h1 class="text-center"><%=  @recipe.name %>
+<% if @recipe.public %>
+    <span class="badge bg-secondary">Public</span>
+    <% else %>
+    <span class="badge bg-secondary">Private</span>
+    <% end %>
+</h1>
+<% if can? :manage, @recipe %>
+<% if @recipe.public %>
+  <%= button_to "Public", update_recipe_path(id: @recipe), method: :patch, class:"btn btn-success btn-sm btn1" %>
+<% else %>
+  <%= button_to "Private", update_recipe_path(@recipe), method: :patch, class:"btn btn-secondary btn-sm btn1" %>
+<% end %>
+<% end %>
+<div class="card w-75 container">
+  <div class="card-body row align-items-between">
+    <div class="col col-md-6 ms-3">
+      <h4 class="fw-bold my-1">Preparation Time: <%= @recipe.preparation_time/60 %> hours <%= @recipe.preparation_time%60 %> minutes</h4>
+      <h4 class="fw-bold my-4">Cooking Time: <%= @recipe.cooking_time/60 %> hours <%= @recipe.cooking_time%60 %> minutes</h4>
+    </div>
+  </div>
+  <div class="row card-body">
+    <div class="card-body col col-md-12">
+      <h4 class="fw-bold my-1">Description</h4>
+      <p class="fs-6">
+        <%= @recipe.description %>
+</div>
+<div class="mt-5 d-flex justify-content-around">
+    <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#recipeModal">
+      Generate shopping list
+    </button>
```

Comment:
- [ ] There seems to be a problem with this feature 🐛 . The shopping list for the recipe and it's inventory is not working as expected; clicking the button shows the modal but the "Generate" button in the modal simply reloads the page instead of generating the shopping list. Please fix this 🙏